docs/README.md

```diff
@@ -38,6 +41,31 @@ Each page within a Syna based website is created using fragments.
 
 <!-- /TOC -->
 
+## Installation
+
+### Adding Syna to your site
+
+You can use Syna by adding it as a submodule to your website repository and pointing the submodule to the latest release.
+This way whenever you want to update the theme you can just pull the updates and checkout to the latest tag.
+
+```bash
+git submodule init # If you haven't initialized before
+git submodule add git@github.com:okkur/syna.git themes/syna
+cd themes/syna
+git checkout v0.11.0 # Latest release as of now is v0.11.0
```

Comment:
We should document the various places where a version string needs to be change within some type of release documentation.

Reply:
I think this is the only place that it's actually needed. Does the config.toml->version do anything? Also there is VERSION file which is I think just to keep a record so it's obvious what version the developer is working on.

Reply:
So 3 different places. Still should be documented.
Theme version might be used for themes.gohugo.io so makes sense to keep up to date.
VERSION might be shown within git UIs such as gitlab and is a good way to keep track.
And the above version in the documentation (once in the starter and one in thre README).

So basically 4 places ^^

Reply:
Docs in a follow on PR. Or just create an issue for now.